StreamChatSwiftUITestsAppTests/Tests/Base TestCase/StreamTestCase.swift

```diff
@@ -14,25 +14,29 @@ class StreamTestCase: XCTestCase {
     var backendRobot: BackendRobot!
     var participantRobot: ParticipantRobot!
     var server: StreamMockServer!
+    var recordVideo = false
 
     override func setUpWithError() throws {
         continueAfterFailure = false
-        server = StreamMockServer()
-        server.configure()
-        server.start(port: in_port_t(MockServerConfiguration.port))
+        startMockServer()
         participantRobot = ParticipantRobot(server)
         backendRobot = BackendRobot(server)
         userRobot = UserRobot(server)
 
         try super.setUpWithError()
+        alertHandler()
         useMockServer()
+        startVideo()
         app.launch()
     }
 
     override func tearDownWithError() throws {
+        attachElementTree()
+        stopVideo()
         app.terminate()
         server.stop()
         server = nil
+        backendRobot.delayServerResponse(byTimeInterval: 0.0)
```

Comment:
What's the idea behind delaying for 0.0?

Reply:
To emulate connection issues, we can `delayServerResponse` in some tests. So we set it back to `0` on `tearDown`.